From e51cae17c9c228d2044ea5984b2f9f92c476b53f Mon Sep 17 00:00:00 2001
From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Date: Thu, 26 Mar 2020 14:28:25 +0200
Subject: [PATCH 76/89] platform/x86: surface3_power: Add missed headers

We obviously are users of bits.h and types.h. Add them to the list.

Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
---
 drivers/platform/x86/surface3_power.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/platform/x86/surface3_power.c b/drivers/platform/x86/surface3_power.c
index 692ba845a1b4..cf08911251f3 100644
--- a/drivers/platform/x86/surface3_power.c
+++ b/drivers/platform/x86/surface3_power.c
@@ -32,11 +32,13 @@
  */
 
 #include <linux/acpi.h>
+#include <linux/bits.h>
 #include <linux/freezer.h>
 #include <linux/i2c.h>
 #include <linux/kernel.h>
 #include <linux/kthread.h>
 #include <linux/slab.h>
+#include <linux/types.h>
 #include <linux/uuid.h>
 #include <asm/unaligned.h>
 
-- 
2.30.2

